deployments/cardano-prod.nix

```diff
@@ -6,13 +6,24 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["env:production"];
 
     # DEVOPS-64: disable log bursting
     services.journald.rateLimitBurst = 0;
 
     # Initial block is big enough to hold 3 months of transactions
     deployment.ec2.ebsInitialRootDiskSize = mkForce 700;
+
+    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.processConfig = ''
+      init_config:
+
+      instances:
+      - name: cardano-node
+        search_string: ['cardano-node']
+        exact_match: False
+        thresholds:
+          critical: [1, 2]
```

Comment:
We were planning to use two nodes for DDOS protection, but specification changed so we can set this back to 1, thanks!